### PR TITLE
fix typos and SUPERUSER_PASSWORD

### PR DESCRIPTION
**Pull request by KagurazakaNyaa**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,23 +4,23 @@ FROM ubuntu:21.10 as base
...
+  libqt5network5 \
+  libqt5sql5 \
+  libqt5sql5-mysql \
+  libqt5sql5-psql \
+  libqt5sql5-sqlite \
+  libqt5xml5 \
+  libqt5dbus5 \
+  ca-certificates \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```
Comment (Krzmbrzl) [inline Dockerfile:23]: This seems to be an unrelated change

Comment (KagurazakaNyaa) [reply Dockerfile:23]: This is vscode's automatic formatting for Dockerfile, which makes the indentation of different parts within the same file consistent.

Comment (Krzmbrzl) [reply Dockerfile:23]: Then please turn that off. Formatting changes should be kept separate from functional changes.

I think at this point we don't have a dedicated formatting guideline for the file, but everyone always reformatting the source file because their editor thinks it likes it better in a different way, will only lead to chaos :shrug: 

You want to propose a reformatting of the file, please do so in a separate commit (and maybe also separate PR) :)

Comment (KagurazakaNyaa) [reply Dockerfile:23]: OK, so should I create a commit to revert this change?

Comment (Krzmbrzl) [reply Dockerfile:23]: Nah, just edit the existing one (rebase) and then force push your branch to update the PR

Comment (Krzmbrzl) [reply Dockerfile]: Thanks :+1:

**Comment by azlux** [comment]
I'm not sure this PR is a good practice, If people forgot to remove then environment variable, you force the config each time.

**Comment by KagurazakaNyaa** [comment]
> I'm not sure this PR is a good practice, If people forgot to remove then environment variable, you force the config each time.

This behavior is what I expected, it ensures the consistency of environment variables and database.
If the user needs to change the superuser password, then modify the environment variables and recreate the container, no longer need to modify through `docker exec`.
